lib/WUI/nhttp/gcode_upload.cpp

```diff
@@ -245,24 +253,35 @@ namespace {
             strlcat(fn, dest_fname, FILE_PATH_BUFFER_LEN - USB_MOUNT_POINT_LENGTH);
         }
 
+        FILE *attempt = fopen(fn, "r");
+        if (attempt) {
+            fclose(attempt);
+            if (overwrite) {
+                int result = remove(fn);
+                if (result == -1) {
+                    switch (errno) {
+                    case EBUSY:
+                        return make_tuple(Status::Conflict, "File is busy");
+                    default:
+                        return make_tuple(Status::NotFound, "Unknown error");
```

Comment:
If it's unknown error, then we probably shouldn't return 404 Not Found, which is _client_ error. This should go to some 5?? error.